README.md

```diff
@@ -17,6 +17,7 @@ optional arguments:
                         output directory
   --skip-dependencies, -x
                         does not check for the dependencies
+  --no-isolation, -n    do not isolate the build in a virtual environment
```

Comment:
We should probably not show it here for maintenance reasons, and move it to the readme docs entirely via https://sphinx-argparse.readthedocs.io/en/stable/

Reply:
I am having some trouble getting it to find the parser, so I am gonna do it in a separate PR.